packages/venia-concept/webpack.config.js

```diff
@@ -1,9 +1,12 @@
-const { configureWebpack } = require('@magento/pwa-buildpack');
+const {
+    configureWebpack,
+    fetcherUtils: { getMediaURL }
+} = require('@magento/pwa-buildpack');
 const { DefinePlugin } = require('webpack');
 const HTMLWebpackPlugin = require('html-webpack-plugin');
-const MediaBackendUrlFetcherPlugin = require('./src/MediaBackendURLFetcherPlugin');
 
 module.exports = async env => {
+    await getMediaURL();
```

Comment:
Do we not need to do anything with the result of this?

I see the promise also sets `global.MAGENTO_MEDIA_BACKEND_URL` ... we probably want to move that out of being a side effect of the "fetcher" and have some code intentionally set that value elsewhere (here?).

Reply:
This is the main thing I'd like to see changed, then 👍

Reply:
The idea was to not clutter exports object with these. But Ill refactor this to have a getter that is pure and a setter that has the global object setting logic.